pkg/apis/proto/daemon/daemon.proto

```diff
@@ -50,4 +68,8 @@ service DaemonService {
   rpc GetBuffer (GetBufferRequest) returns (GetBufferResponse) {
     option (google.api.http).get = "/api/v1/pipelines/{pipeline}/buffers/{buffer}";
   };
+
+  rpc GetVertexMetrics (GetVertexMetricsRequest) returns (GetVertexMetricsResponse) {
+    option (google.api.http).get = "/api/v1/namespace/{namespace}/pipelines/{pipeline}/vertex/{vertex}";
```

Comment:
Is `/api/v1/namespace/{namespace}/pipelines/{pipeline}/vertices/{vertex}/metrics` working?